packages/codegen-ui/lib/generate-form-definition/helpers/form-field.ts

```diff
@@ -214,7 +214,7 @@ export function getFormDefinitionInputElement(
         props: {
           label: config.label || baseConfig?.label || FORM_DEFINITION_DEFAULTS.field.inputType.label,
           descriptiveText: config.inputType?.descriptiveText ?? baseConfig?.inputType?.descriptiveText,
-          placeholder: config.inputType?.placeholder || baseConfig?.inputType?.placeholder,
+          placeholder: config.inputType?.placeholder || baseConfig?.inputType?.placeholder || 'Please select an option',
```

Comment:
does this need i18n?

Reply:
Yea though I think we'll need to address this as a follow up in a larger sense as most text does not support i18n today